test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/AbstractCloudIntegrationTest.java

```diff
@@ -52,6 +52,7 @@
     protected static final String USER_YODA = "yoda";
 
     protected static final String USERTASK_PROCESS_ID = "definition-project.usertask";
+    protected static final String UPDATED_USERTASK_PROCESS_ID = "definition-project.updated-usertask";
```

Comment:
What's the difference between USERTASK_PROCESS_ID and UPDATED_USERTASK_PROCESS_ID?

Reply:
USERTASK_PROCESS_ID is process id from definition project 1.0.0
UPDATED_USERTASK_PROCESS_ID is process id from definition project 1.0.1

I created it so I am able to verify that the call reached container with definition project 1.0.1